docker-compose.extended.yml

```diff
@@ -50,7 +52,7 @@ services:
                     - 'node.platform.os == linux'
 
     queue-worker:
-        image: functions/queue-worker:0.1.1
+        image: functions/queue-worker:latest-dev
```

Comment:
oops.. ^

Reply:
you ain't seen nothing right, this didn't happen *cough*

Reply:
Fixed in aba8850